pkg/vadmin/install_packages_vc.go

```diff
@@ -0,0 +1,68 @@
+/*
+ (c) Copyright [2021-2023] Open Text.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package vadmin
+
+import (
+	"context"
+
+	vops "github.com/vertica/vcluster/vclusterops"
+	"github.com/vertica/vcluster/vclusterops/vstruct"
+	"github.com/vertica/vertica-kubernetes/pkg/net"
+	"github.com/vertica/vertica-kubernetes/pkg/vadmin/opts/installpackages"
+)
+
+// InstallPackages will install all packages under /opt/vertica/packages where Autoinstall is marked true
+func (v *VClusterOps) InstallPackages(_ context.Context, opts ...installpackages.Option) error {
+	v.setupForAPICall("InstallPackages")
+	defer v.tearDownForAPICall()
+	v.Log.Info("Starting vcluster InstallPackages")
+
+	// get install_packages options
+	s := installpackages.Parms{}
+	s.Make(opts...)
+
+	// call vcluster-ops library to install packages
+	vopts := v.genInstallPackagesOptions(&s)
+	status, err := v.VInstallPackages(&vopts)
```

Comment:
Do we print out the status in the log at all for the successful case?

Reply:
will add that